packages/melos/lib/src/common/intellij_project.dart

```diff
@@ -227,11 +229,11 @@ class IntellijProject {
   }
 
   Future<void> clean() async {
-    final runConfigurationsDirectory =
-        Directory(joinAll([pathDotIdea, 'runConfigurations']));
-    if (runConfigurationsDirectory.existsSync()) {
-      await Directory(joinAll([pathDotIdea, 'runConfigurations']))
-          .delete(recursive: true);
+    final melosConfigurations =
```

Comment:
CI is currently failing when the directory doesn't exist, I would suggest wrapping this function body in a:

```dart
if (Directory(joinAll([pathDotIdea, 'runConfigurations'])).existsSync()) {
  // TODO Do your glob and delete melos files
}
```

Reply:
Thanks! I fixed it.